packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -535,30 +535,60 @@ class SelectableRegionState extends State<SelectableRegion> with TextSelectionDe
   }
 
   void _initMouseGestureRecognizer() {
-    switch (defaultTargetPlatform) {
-      case TargetPlatform.android:
```

Comment:
nice clean up. I wonder why we have the switch case before

Reply:
Yeah I think I just forgot to remove it.